src/areaCalculator/Main.java

```diff
@@ -1,8 +1,37 @@
 package areaCalculator;
 
+class Rectangle {
+    private double lengthA;
```

Comment:
What is A? What is B? 
I think names could be more accurate. 
Remember how you call those two things mathematically

Reply:
Changed to length and width